app/models/user.rb

```diff
@@ -80,4 +80,16 @@ def remove_pregnancy(pregnancy)
     pregnancies.delete pregnancy
     reload
   end
+
+  def clear_call_list
+    pregnancies.each do |p|
+      pregnancies.delete(p) if recently_reached?(p)
+    end
+  end
+
+  private
+
+  def recently_reached?(preg)
```

Comment:
Also a +1 for changing the name of them both to `*_by_user` I think! My instinct is that `recently_called/reached` could mean that either someone @ dcaf generally called them also, so I think I'd err on the side of specific. Yr thoughts @ajohnson052 ?